common/pom.xml

```diff
@@ -19,97 +19,151 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>eu.seaclouds-project</groupId>
+        <artifactId>platform</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
 
-  <parent>
     <groupId>eu.seaclouds-project</groupId>
-    <artifactId>platform</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
-  </parent>
+    <artifactId>common</artifactId>
+    <packaging>jar</packaging>
+    <description>SeaClouds common module</description>
+
+    <properties>
+        <spring.version>4.1.4.RELEASE</spring.version>
+        <commons-lang.version>3.3.2</commons-lang.version>
+        <validation-api.version>1.1.0.Final</validation-api.version>
+    </properties>
+
+    <repositories>
+        <repository>
```

Comment:
I think it may be a good idea if the issue about mvn version is commented on the Readme. Indicating that to build this project is required at least mvn 3.2.5